core/src/main/resources/jenkins/security/apitoken/LegacyApiTokenAdministrativeMonitor/manage.jelly

```diff
@@ -35,7 +35,8 @@ THE SOFTWARE.
                 <p>${%recommendationMessage}</p>
                 
                 <j:set var="userList" value="${it.impactedUserList}"/>
-                <table class="pane bigtable align-th-left">
+                <table class="pane jenkins-table">
+                <thead>
```

Comment:
I may have mentioned it before, but wouldn't it be more future proof to make these into Jelly tags we can rewrite as needed for future updates? This will be getting copied & pasted all over the place otherwise.

Reply:
it could possibly be added now

something like?
```
l:table
l:tableHeader
l:tableBody
l:tableRow
l:tableFooter
```

Reply:
I think it would be great!

Reply:
Looks good to me!

Reply:
you up for trying that @NotMyFault? could be done in this PR as an example?

Reply:
Sounds like a great idea! Though unfortunately, I don't have as much spare time as I'd love to have to work on features like that, and I didn't work much with Jelly before, so I'd take me some time to get into it 😅

Reply:
i can prob push something in to your branch tomorrow or next week if that's okay?

Reply:
For sure, go ahead!

Reply:
Yes, using some tags would be very helpful. 

(I think it might even make sense to go another step deeper and replace the whole page with a template view that contains the header and the table and takes care of the  responsiveness. And the actual component just provides the data. I used that approach in the bootstrap and [data-tables-api-plugin](https://github.com/jenkinsci/data-tables-api-plugin#tables-with-dynamic-model-based-content) and my plugin views only provide the Java based object model, see [Warnings Issues table](https://github.com/jenkinsci/warnings-ng-plugin/blob/bfef1b523e09fbd654a219f0e058464570ca0101/plugin/src/main/resources/io/jenkins/plugins/analysis/core/model/IssuesDetail/filled.jelly#L111)). So when I hopefully find some time to apply the new tables styles to the bootstrap and data-tables plugins then all plugins that use them will get those changes without additional work.)